src/plugins/data/common/search/aggs/buckets/_terms_other_bucket_helper.ts

```diff
@@ -72,8 +73,8 @@ const getAggResultBuckets = (
       }
     }
   }
-  if (responseAgg[aggWithOtherBucket.id]) {
-    return responseAgg[aggWithOtherBucket.id].buckets;
+  if (responseAgg?.[aggWithOtherBucket.id]) {
+    return (responseAgg[aggWithOtherBucket.id] as any).buckets;
```

Comment:
Introduced response typing, but I don't want to broaden the refactoring of this code